include/wpe-fdo/fdo-egl.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2017, 2018 Igalia S.L.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+ * PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifdef __WEBKIT_WEB_EXTENSION_H__
```

Comment:
Wrong macro?

Reply:
The goal there is to prevent these files from being included by mistake in a web extension. I presume that would always be a mistake, right?

Reply:
Yes, it's just an odd thing to be preventing.

Reply:
@zdobersek I can remove it if you like, but I can pretty much guarantee that confused bug reports will result, so I would recommend keeping it. With WebKitGTK+, we even get bug reports from people confused by the error message that the headers can't be included together. Having no error message will be worse. Anyway, whatever you prefer is fine with me.

Reply:
Note also: https://github.com/WebPlatformForEmbedded/WPEBackend/commit/2bf3c3ae5f56d4d7383601f27c30f98cbe758a7d included the same checks